### PR TITLE
Update screen size info after orientation change on Android/iOS

### LINKED ISSUE #4992: CollectionView doesn't resize on orientation change
### Description
When the orientation of the device changes from Landscape to Portrait the CollectionView doesn't stretch to fit the bottom of the screen, instead the CollectionView keeps it's original height leaving white space at the bottom of the screen. If you start in Portrait all is fine, though it looks like the Collection View doesn't shrink it's height when switching to Landscape.

### Steps to Reproduce

1. Download the Collection View sample from https://github.com/pauldipietro/CollectionViewSample
2. Build and run the sample with the device in Landscape
3. Rotate the device to Portrait

### Expected Behavior

CollectionView will increase it's height to fill the screen

### Actual Behavior
CollectionView height remains unchanged leaving whitespace at the bottom of the screen

### Basic Information

- Version with issue: Xamarin.Forms 4.0.0.8055-pre1
- Last known good version: None
- IDE: Visual Studio 2017 15.8.5
- Platform Target Frameworks:
  - Android: 8.1
- Android Support Library Version: 28
- Nuget Packages: All in the Sample linked
- Affected Devices: Android Simulator - Oreo

### Screenshots
image
image

### Reproduction Link
https://github.com/pauldipietro/CollectionViewSample

### PR DESCRIPTION
**Pull request by hartez**
### Description of Change ###

Updates the Device.Info screen information on Android/iOS when the orientation of the device changes.

### Issues Resolved ### 

- fixes #4992

### API Changes ###

 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
In Control Gallery, navigate to Issue 4992 and follow the instructions.

(Can't automate the test, because the app has to start in landscape mode to reproduce the issue.)

### PR Checklist ###

- [ ] Has automated tests 
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

**Comment by hartez** [comment]
iOS 10 test failure is a known unrelated problem.

File: Xamarin.Forms.Platform.Android/Forms.cs [hunk 1]
```diff
@@ -304,21 +304,15 @@ class AndroidDeviceInfo : DeviceInfo
 		{
 			bool _disposed;
 			readonly Context _formsActivity;
-			readonly Size _pixelScreenSize;
-			readonly double _scalingFactor;
+			private Size _scaledScreenSize;
```
Comment (samhouts) [inline Xamarin.Forms.Platform.Android/Forms.cs]:
```suggestion
			Size _scaledScreenSize;
```

File: Xamarin.Forms.Platform.MacOS/MacDeviceInfo.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using AppKit;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Platform.macOS
+{
+	internal class MacDeviceInfo : DeviceInfo
+	{
+		private Size _pixelScreenSize;
```
Comment (samhouts) [inline Xamarin.Forms.Platform.MacOS/MacDeviceInfo.cs]:
```suggestion
		Size _pixelScreenSize;
```

File: Xamarin.Forms.Platform.iOS/IOSDeviceInfo.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+using Foundation;
+using UIKit;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Platform.iOS
+{
+	internal class IOSDeviceInfo : DeviceInfo
+	{
+		private Size _pixelScreenSize;
```
Comment (samhouts) [inline Xamarin.Forms.Platform.iOS/IOSDeviceInfo.cs]:
```suggestion
		Size _pixelScreenSize;
```

**Comment by hartez** [comment]
Android API 19 failures are a known unrelated issue: https://github.com/xamarin/Xamarin.Forms/issues/6061

**Pull request merged**